### PR TITLE
Show unfilled in estimate

### PR DESCRIPTION
**Pull request by grod220**
Closes: https://github.com/penumbra-zone/web/issues/566

Here's a normal swap estimate:

Screenshot 2024-03-11 at 2 27 13 PM

With this PR, if a part of the estimate is expected to be unfilled, we'll show labels:

Screenshot 2024-03-11 at 2 26 55 PM

### CODE REVIEW

File: packages/types/src/index.ts [hunk 1]
```diff
@@ -19,8 +19,6 @@ export * from './querier';
 export * from './amount';
 export * from './chain';
 export * from './jsonified';
-export * from './schemas';
-export * from './type-predicates';
```
Comment (grod220) [inline packages/types/src/index.ts:23]: Cleaning up a number of unused zod types that were before the Getter days

**Review by Valentine1898 — APPROVED** [review]
LGTM! 

But I think we will need to improve the UX for swaps in the future. 
Having unfilled amount leads to additional action in the transaction and thus higher transaction cost, instead it would be better to ask the user to change the amount to the maximum amount that can be filled.

**Comment by hdevalence** [comment]
> Having unfilled amount leads to additional action in the transaction and thus higher transaction cost, instead it would be better to ask the user to change the amount to the maximum amount that can be filled.

Agree on needing to improve swap UX in the future, but this is not the case: every `SwapClaim` produces two output notes, there's no difference to the user whether the swap was completely filled or not.

**Comment by Valentine1898** [comment]
> > Having unfilled amount leads to additional action in the transaction and thus higher transaction cost, instead it would be better to ask the user to change the amount to the maximum amount that can be filled.
> 
> Agree on needing to improve swap UX in the future, but this is not the case: every `SwapClaim` produces two output notes, there's no difference to the user whether the swap was completely filled or not.

Right, my bad

**Pull request merged**